FlowCrypt/Core/Core.swift

```diff
@@ -21,6 +23,8 @@ enum CoreError: Error, Equatable {
         switch errorType {
         case "format": self = .format(coreError.error.message)
         case "key_mismatch": self = .keyMismatch(coreError.error.message)
+        case "no_mdc": self = .noMDC(coreError.error.message)
+        case "bad_mdc": self = .badMDC(coreError.error.message)
         default: return nil
```

Comment:
Need to check in another PR that all errors are represented - at least it's missing need_passphrase and bad_passphrase

Reply:
It's already handled when user opens message screen, but I may add it anyway